draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -187,29 +187,26 @@
+      <t>As IS-IS is deployed in greater scale both in the number of nodes in an
+      area and in the number of neighbors per node, the impact of the historic
       flooding rates becomes more significant. Consider the bringup or failure
       of a node with 1000 neighbors. This will result in a minimum of 1000 LSP
-      updates. At a typical LSP flooding rate used in many deployments today
+      updates. At typical LSP flooding rates used today
       (33 LSPs/second), it would take 30+ seconds simply to send the updated
       LSPs to a given neighbor. Depending on the diameter of the network,
       achieving a consistent LSDB on all nodes in the network could easily
-      take a minute (or more).</t>
+      take a minute or more.</t>
 
-		<t>In addition, one single event in the network can require the flooding of multiple LSPs. The typical case is a node failure which requires the flooding of at least one LSP per neighbor of the failed node. If a node has N IGP neighbors, the network won't be able to converge to the new topology until all N LSPs are received by all nodes. Hence there is a need to be able to quickly exchange N LSPs.</t>
-
-      <t>Increasing LSP flooding rate therefore becomes an essential element
+      <t>Increasing the LSP flooding rate therefore becomes an essential element
       of supporting greater network scale.</t>
 
-			<t>Increasing LSP flooding rate does not compete with proposed extensions to reduce LSP flooding traffic by reducing the flooding topology such as Mesh Groups ( <xref target="RFC2973"/> )or 
-				<xref target="I-D.ietf-lsr-dynamic-flooding"/>
-. On the contrary, this extension complements those proposals. Indeed reducing the flooding topology does not reduce the size of the LSDB or the total number of LSPs to exchange between two nodes. So increasing the overall flooding speed can be beneficial for nodes implementing dynamic flooding. The reverse is also true: as dynamic flooding reduces the number of neighbors with flooding enabled, this allows nodes implementing the flooding parameter extensions to focus their flooding resources on those neighbors by sending better parameters to the selected flooding nodes and worse parameters to non-selected flooding nodes.
+			<t>Improving the LSP flooding rate is complementary to other proposed extensions to reduce LSP flooding traffic by reducing the flooding topology such as Mesh Groups <xref target="RFC2973"/> or Dynamic Flooding
+				<xref target="I-D.ietf-lsr-dynamic-flooding"/>. Reducing the flooding topology does not reduce the size of the LSDB or the total number of LSPs to be exchanged between two nodes, so increasing the overall flooding speed can be beneficial for nodes implementing Dynamic Flooding. The reverse is also true: Dynamic Flooding reduces the number of flooding neighbors and can be further enhanced by selecting the topologically appropriate neighbors that support improved flooding performance.
```

Comment:
"Improving the LSP flooding rate is complementary to protocol
   extensions that reduce LSP flooding traffic by reducing the flooding
   topology such as Mesh Groups [RFC2973] or Dynamic Flooding
   [I-D.ietf-lsr-dynamic-flooding].  Reduction of the flooding topology does
   not alter the number of LSPs required to be
   exchanged between two nodes, so increasing the overall flooding speed
   is still beneficial when such extensions are in use. It is also
   possible that the flooding topology can be reduced in ways that
   prefer the use of neighbors that support improved flooding performance."

NOTE: Although dynamic flooding is far superior to mesh groups, the current text starts by mentioning both extensions and then specifically calls out dynamic flooding when the text is equally applicable to mesh groups. I tried to address this in my revised text.

Reply:
```suggestion
<t>Improving the LSP flooding rate is complementary to protocol
extensions that reduce LSP flooding traffic by reducing the flooding
topology such as Mesh Groups <xref target="RFC2973"/> or Dynamic Flooding
				<xref target="I-D.ietf-lsr-dynamic-flooding"/>. Reduction of the flooding topology does
not alter the number of LSPs required to be
exchanged between two nodes, so increasing the overall flooding speed
is still beneficial when such extensions are in use. It is also
possible that the flooding topology can be reduced in ways that
prefer the use of neighbors that support improved flooding performance.
```

Reply:
WFM

Reply:
Text accepted.

Authorship note: github is REALLY awkward with really long lines.  After all it's meant for code.  Please try to reduce text width when possible.